tools/harness-thci/ARM.py

```diff
@@ -1556,25 +1556,49 @@ def startNativeCommissioner(self, strPSKc='GRLpassWord'):
         pass
 
     def startCollapsedCommissioner(self):
-        pass
+        """start openthread stack"""
```

Comment:
Could we add more description about input parameters and return value? Maybe we can refer to "IThci.py" file in harness install directory. It would be easier for people to understand the functionality of each method.